src/plugins/data_table/filter_view.js

```diff
@@ -68,43 +76,79 @@ var DataTableFilter = (function(options) {
         this.filter.value = filterInput.value;
         this.parentView._toggleIcon(true);
       }
+    },
+
+    afterRender: function() {
+      this.$('.btn-filter').click(this.parentView._onFilterClick.bind(this.parentView));
+      this.$('.btn-clear').click(this.parentView._onClearClick.bind(this.parentView));
     }
   });
 
   var NumericFilterMenu = DataTableFilterMenu.extend({
-    tagName: "ul",
-
     menuTemplate: _.template('\
-        <li> &gt; <input data-numeric-filter-name="gt" class="filter-numeric filter-numeric-greater" type="text" /></li> \
-        <li> &lt; <input data-numeric-filter-name="lt" class="filter-numeric filter-numeric-less" type="text"/></li> \
-        <li> = <input data-numeric-filter-name="eq" class="filter-numeric filter-numeric-equal" type="text" /></li> \
-      ', null, DEFAULT_JST_DELIMS),
+      <div class="filterMenu filterMenu-numeric">\
+        <div class="filter-text">Show items with value that:</div>\
+        <select class="filter-type form-control" data-filter-id="first-filter">\
+          <option selected value="gt">is greater than</option> \
+          <option value="lt">is less than</option> \
+          <option value="eq">is equal to</option> \
+        </select> \
+        <input id="first-filter" class="filter-value form-control" type="text" data-filter-type="gt"  /> \
```

Comment:
do you need the `id` attrs here and in the second filter? Hopefully not.. if so let's make them classes instead